operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -2503,6 +2567,23 @@ memcachedExporter:
   #   memcached.tls.server-name: memcached
   extraArgs: {}
 
+  # -- Base config file for memcached_exporter as per https://github.com/prometheus/exporter-toolkit/blob/master/docs/web-configuration.md
```

Comment:
let's mention somewhere here that we expect the secret of config map to have the key `web-config.yaml`